js/main.js

```diff
@@ -55,4 +55,4 @@ let objVar = '';
 for (let i = 0; i < bookObj.length; i += 1) {
   objVar += `<li><p><span>${bookObj[i].bookTitle}</span> by <span>${bookObj[i].arthur}</span> </p> <button id=${bookObj[i].id.toString()} onClick="removeBook(${bookObj[i].id})">Remove</button></li>`;
 }
-bookList.innerHTML = objVar;
+bookList.innerHTML = objVar;
```

Comment:
- [ ] @David-Lanzz and @stephenkati great work on your application, however, you forgot to implement the classes for add and delete functions, also ensure your arrays are inside classes this is a project requirement
![image](https://user-images.githubusercontent.com/61408860/218801882-0ed9bd29-a296-4303-8d65-4b91e771e7d9.png)